app/handlers/test_set.py

```diff
@@ -105,6 +109,25 @@ def _post(self, *args, **kwargs):
 
         return response
 
+    def _update_test_suite_add_test_set_id(self, set_oid, suite_oid, suite_name):
+        """Execute the async task to add the test set ID onto the test suite.
+
+        :param set_oid: The ID of the test set.
+        :type set_oid: bson.objectid.ObjectId
+        :param suite_oid: The ID of the test suite.
+        :type suite_oid: bson.objectid.ObjectId
+        :param suite_name: The name of the test suite.
+        :type suite_name: str
+        """
+        taskq.update_test_suite_add_test_set_id.apply_async(
+            [
+                set_oid,
+                suite_oid,
+                suite_name,
+                self.settings["dboptions"], self.settings["mailoptions"]
```

Comment:
Since this is new code, and passing the database/mail options is kind of an hold and bad habit, can you please remove it from here?

You can still access the database/mail options directly from the celery app, and we avoid passing connection parameters to celery nodes that can potentially be on a different network.